### ISSUE TITLE
Gson OSGi metadata claims support for JavaSE-1.5 but target generation is 1.6

### ISSUE THREAD

**Issue by briandealwis** [google/gson#1602]
Although the OSGi metadata claims Gson will work with Java 1.5 or greater:
https://github.com/google/gson/blob/47b28d99a17d492b37fcdd0fc41290419a010a0f/gson/bnd.bnd#L6

the compilation configuration sets the target as 1.5 and the classes themselves are Java 1.6:
https://github.com/google/gson/blob/47b28d99a17d492b37fcdd0fc41290419a010a0f/pom.xml#L89-L95

```sh
$ file gson/target/classes/com/google/gson/Gson.class
gson/target/classes/com/google/gson/Gson.class:                       compiled Java class data, version 50.0 (Java 1.6)
```

And that looks to have been a decision made a while ago #790.

**Comment by elharo** [comment]
fixed by #1603

**Comment by chanseokoh** [comment]
Please close this.
